### PR TITLE
Frozen Hell 1.8

### PR DESCRIPTION
**Pull request by skyem123**

### CODE REVIEW

File: descentfrozenhell/config/immersiverailroading.cfg [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            purified_water
+        >
+        
+    }
+    
+    damage {
+        # Trains should break block
+        B:TrainsBreakBlocks=false
+        
+        # Clear blocsk in creative mode when placing tracks
```
Comment (NickStalburg) [inline descentfrozenhell/config/immersiverailroading.cfg:123]: Fix the spelling of 'blocks'

**Review by NickStalburg — CHANGES_REQUESTED** [review]

**Review by NickStalburg — APPROVED** [review]
Yes all looks good.

**Review by skyem123 — COMMENTED** [review]
All done!

**Review by ThePiGuy24 — APPROVED** [review]
it looks like it probably wont break everything, good enough.

**Review by CitadelCore — APPROVED** [review]

**Comment by CitadelCore** [comment]
ISC approved, ready for merge

**Comment by skyem123** [comment]
Once it passes CI, I will merge it.

**Pull request merged**